scripts/globals/effects/evasion_down.lua

```diff
@@ -3,28 +3,36 @@
 --  EFFECT_EVASION_DOWN
 --
 -----------------------------------
-
 require("scripts/globals/status");
 
 -----------------------------------
 -- onEffectGain Action
 -----------------------------------
-
 function onEffectGain(target,effect)
+    if (target:getMod(MOD_EVA) - effect:getPower() < 0) then
+        effect:setPower(target:getStat(MOD_EVA));
+    end
     target:addMod(MOD_EVA,-effect:getPower());
-end;
+end
 
 -----------------------------------
 -- onEffectTick Action
 -----------------------------------
-
 function onEffectTick(target,effect)
-end;
+    -- If this is Feint, restore 10 evasion every tick
```

Comment:
how do we know this is from feint?

Reply:
I guess the fact that it has a tick rate at all.. nevermind, seems well enough (I can't think of a reason why any other evasion down would have a tick rate that isn't shedding the effect)

Reply:
Yes, exactly. It's the only thing that ticks. I can update the comment to be a little more specific.